noxfile.py

```diff
@@ -658,25 +658,16 @@ def system_prerelease(session: nox.sessions.Session):
 
 @nox.session(python=SYSTEM_TEST_PYTHON_VERSIONS)
 def notebook(session: nox.Session):
-    GOOGLE_CLOUD_PROJECT = os.getenv("GOOGLE_CLOUD_PROJECT")
-    if not GOOGLE_CLOUD_PROJECT:
-        session.error(
-            "Set GOOGLE_CLOUD_PROJECT environment variable to run notebook session."
-        )
-
-    session.install("-e", ".[all]")
-    session.install(
-        "pytest",
-        "pytest-xdist",
-        "pytest-retry",
-        "nbmake",
-        "google-cloud-aiplatform",
-        "matplotlib",
-        "seaborn",
-    )
-
-    notebooks_list = list(Path("notebooks/").glob("*/*.ipynb"))
-
+    folders = [
```

Comment:
I worry about making this an allowlist. Seems like the default should be to run notebooks in the notebooks session

Reply:
Okay, we can do a folders_denylist then if that's OK (since we will want to filter out experimental and benchmark notebooks by folder).

Reply:
Yes, I'd prefer a denylist. Alternatively, I'd be curious to see exploring just using regular pytest / python files for our benchmark runs.

Reply:
Per below, we're switching to regular python files for benchmarks.